### PR TITLE
[DEV-557] changed crypto route fee if a ref was used

### PR DESCRIPTION
**Pull request by Krysh90**
#### Pre-Release

- [x] Check migrations
  - No database related infos (sqldb-xxx)
  - Impact on GS (new/removed columns)
- [ ] Check for linter errors (in PR)
- [ ] Test basic user operations
  - Login/logout
  - Registration
  - Update user data
  - Create/delete payment routes

#### Post-Release

- Test basic user operations
- Monitor application insights log

### CODE REVIEW

File: src/config/config.ts [hunk 1]
```diff
@@ -205,6 +205,7 @@ export class Configuration {
 
   crypto = {
     fee: 0.012,
+    feeUsedRef: 0.011,
```
Comment (davidleomay) [inline src/config/config.ts:208]: I would propose to switch to a refBonus of 0.1% instead. So the fee, when using a ref code would be `crypto.fee - crypto.refBonus` (see buy fee).

Comment (Krysh90) [reply src/config/config.ts]: resulting from discussion yesterday evening, changed calculation to be similar to buy routes and renamed config entry to `refBonus` and added `refBonus` to be delivered with crypto routes

File: src/user/models/user/user.service.spec.ts [hunk 2]
```diff
@@ -146,4 +146,22 @@ describe('UserService', () => {
...
+    setup(AccountType.PERSONAL);
+
+    await expect(service.getUserCryptoFee(1)).resolves.toStrictEqual({ fee: 1.2, refBonus: 0 });
+  });
+
+  it('should return a fee of 1.1 and refBonus of 0.1 for crypto routes, if ref was used', async () => {
+    setup(AccountType.PERSONAL, undefined, undefined, '000-001');
+
+    await expect(service.getUserCryptoFee(1)).resolves.toStrictEqual({ fee: 1.1, refBonus: 0.1 });
+  });
```
Comment (davidleomay) [inline src/user/models/user/user.service.spec.ts:166]: Add a unit test for individual crypto fee?

Comment (Krysh90) [reply src/user/models/user/user.service.spec.ts:166]: fair enough. will add one

**Review by davidleomay — APPROVED** [review]

**Pull request merged**